metatron-app/metatron-discovery
# F#1350 download file of imported dataset

### Description

Download file of imported dataset(only Upload Uri type)

**Related Issue** : 

#1350 

### How Has This Been Tested?

1. menu> Data Preparation > Dataset
2. Click Imported dataset(upload, file type)
3. Click URI and check download

### Types of changes



### Checklist:



### Additional Context
